test/end-to-end/test_service_pids_written_to_file_using_old_launcher.ps1

```diff
@@ -21,7 +21,7 @@ Describe "Using a Launcher that cannot provide service PIDs" {
         Restart-Supervisor
         Wait-Process redis-server -Timeout 10
         $newSupProc = Get-Process hab-sup
-        $newRedisProc = Get-Process redis-server
+        $newRedisProc = Get-Process redis-server*
```

Comment:
Do we know what port redis is going to start on? Should we hardcode the value we expect?

Reply:
good call. Just added that.